### PR TITLE
Update a bunch of entries.

### PR DESCRIPTION
**Pull request by bangerth**
This is the replacement for #218. It contains everything that was in #218, plus a unicode fix.

### CODE REVIEW

**Review by masterleinad — APPROVED** [review]

File: publications-2019.bib [hunk 1]
```diff
@@ -312,21 +313,27 @@ @article{Wick2019OptimizationWN
...
 }
 
-@Article{AE19,
-  author =       {J. C. Araujo Cabarcas and C. Engstr{\"o}m},
-  title =        {Removal of spurious solutions encountered in Helmholtz scattering resonance computations in $R^d$},
-  note =      {submitted},
-  year =         {2019},
-  url =       {https://arxiv.org/abs/1904.08812}
+@misc{cabarcas2019spurious,
+    title={On spurious solutions encountered in Helmholtz scattering resonance computations in R^d with applications to nano-photonics and acoustics},
```
Comment (tjhei) [inline publications-2019.bib:331]: You need to escape the R^d

Comment (bangerth) [reply publications-2019.bib]: Oof. I spent half an hour on trying to find this problem. The CI output wasn't useful, nor were my local tests -- they complained about a unicode character, which turned out to not actually exist...

**Comment by bangerth** [comment]
OK, seems to work now!

**Comment by bangerth** [comment]
I'll take the liberty to merge since it wasn't my stuff to begin with.

@dklong-csu -- now update your local clone of the repository with the state from this master repo and extract a pull request from your subsequent changes.

**Pull request merged**